cmake/Modules/CodeCoverage.cmake

```diff
@@ -192,23 +192,26 @@ function(ADD_CODE_COVERAGE)
       set(OMIT_FLAGS --omit ${Coverage_EXCLUDES})
     endif()
 
+    add_custom_target(${Coverage_NAME}_nosetests_py
+        COMMAND ${PYTHON_COVERAGE_PATH} report --include "*${REAL_SOURCE_DIR}*" ${OMIT_FLAGS} || echo "WARNING: No python nosetests report to output"
+        COMMAND ${PYTHON_COVERAGE_PATH} xml  -o ${Coverage_NAME}_nosetests_python.xml --include "*${REAL_SOURCE_DIR}*" ${OMIT_FLAGS} || echo "WARNING: No python nosetests xml to output"
+        WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
+        DEPENDS ${Coverage_NAME}_cpp
```

Comment:
Why do you need to run this command after cpp coverage test?

Reply:
I'm worried about file crash between each coverage process.
nosetests and roslaunch python coverage have file name crash, so we need to run one by one.
But c++, maybe we don't need to do that.
I'm not pretty sure, so I move to output files to `cpp_coverage` directory in build.